hs_access_control/models.py

```diff
@@ -1267,14 +1267,15 @@ def can_change_resource_flags(self, this_resource):
 
         This is not enforced. It is up to the programmer to obey this restriction.
 
-        This is not subject to immutability. Ar present, owns_resource -> can_change_resource_flags. If we made it subject to immutability, no resources could be made not immutable again. 
+        This is not subject to immutability. Ar present, owns_resource -> can_change_resource_flags. If we made it subject to immutability, no resources could be made not immutable again.
+        However, it should account for whether a resource is published, and return false if a resource is published
         """
         if __debug__:  # during testing only, check argument types and preconditions
             assert isinstance(this_resource, BaseResource)
 
         if not self.user.is_active: raise PermissionDenied("Requesting user is not active")
 
-        return self.user.is_superuser or self.owns_resource(this_resource)
+        return self.user.is_superuser or (not this_resource.raccess.published and self.owns_resource(this_resource))
```

Comment:
I concur that superusers need to be able to fix things - especially at the beginning.